src/widgets/label.rs

```diff
@@ -175,16 +175,16 @@ impl Widget for Label {
         } else {
             ""
         };
-        let selectable = if self.state.selectable() {
-            "selectable"
+        let selectable_class = if self.state.unselectable() {
```

Comment:
"unselectable" and "selectable" class ? Just want confirmation ;)

Reply:
it's a not nice name for the var, it should be something like `selectable_status_css_class`, but it's too long for my taste :)
I don't like to use the name `unselectable` because the vaule should be `""` or `"selectable"`, as the css class in app.scss is `.selectable`, to override the default role.

Reply:
i meant you have something like
```rust
if ... {
    "selectable"
} else {
    "unselectable"
}

Reply:
i am talking about the css class :)

Reply:
i meant you have something like
```rust
if ... { "selectable" } else { "unselectable" }
you should either have
```rust
if ... { "selectable" } else{ "" } (or if and else reversed)
```
or 
```rust
if ... { "unselectable" } else { "" } (or if and else reversed)
```